docs/source/api/_devices.rst

```diff
@@ -201,6 +201,7 @@ Other Support
     ~apstools.devices.flyer_motor_scaler.SignalValueStack
     ~apstools.devices.srs570_preamplifier.SRS570_PreAmplifier
     ~apstools.devices.struck3820.Struck3820
+    ~apstools.devices.delay.DG645Delay
```

Comment:
Also add (further down in this file) lines for this module similar to: https://github.com/BCDA-APS/apstools/blob/dd3d9949154e54911dbc361942bbf9846161f5b3/docs/source/api/_devices.rst?plain=1#L266C1-L270

Reply:
```py
.. automodule:: apstools.devices.dict_device_support
    :members:
    :private-members:
    :show-inheritance:
    :inherited-members:
```